app/models/manageiq/providers/azure/manager_mixin.rb

```diff
@@ -166,16 +173,14 @@ def raw_connect(client_id, client_key, azure_tenant_id, subscription, proxy_uri
 
       ::Azure::Armrest::Configuration.log = $azure_log
 
-      connection_rescue_block do
```

Comment:
The rescue block seems to make more sense for `self.verify_credentials` where we need to format the response to be shown on the UI, raw_connect shouldn't mess with exceptions IMO (only used elsewhere by the instance-level connect)